src/hooks/useFunkitBuyNobleUsdc.ts

```diff
@@ -0,0 +1,54 @@
+import { useCallback, useMemo } from 'react';
+
+import { useFunkitCheckout } from '@funkit/connect';
+
+import { DialogTypes } from '@/constants/dialogs';
+
+import { useAppDispatch } from '@/state/appTypes';
+import { closeDialog, openDialog } from '@/state/dialogs';
+
+import { useAccounts } from './useAccounts';
+import { useFunkitThemeListener } from './useFunkitThemeListener';
+
+const TOKEN_SYMBOL = 'USDC';
+const TOKEN_ICON_SRC = '/currencies/usdc.png';
+const TOKEN_CONTRACT_CHAIN_ID = '1511490300';
```

Comment:
What chain is this, I couldn't identify it base on the id 😅

Reply:
This is funkit's internal chainId reference to dydx-mainnet-1 :)